src/components/video-box/VideoBox.tsx

```diff
@@ -1,20 +1,26 @@
 import { FC } from 'react'
 import Box from '@mui/material/Box'
-
 import titleBar from '~/assets/img/guest-home-page/title-bar.png'
 import { styles } from '~/components/video-box/VideoBox.styles'
+import videoImg from '~/assets/img/tutor-profile-page/presentationVideoImg.png'
+import VideoPlayer from '../video-player/VideoPlayer'
 
 interface VideoBoxProps {
   video: string
+  videoPreview: boolean
 }
 
-const VideoBox: FC<VideoBoxProps> = ({ video }) => {
+const VideoBox: FC<VideoBoxProps> = ({ video, videoPreview }) => {
```

Comment:
We should add a default value `videoPreview = true`. Otherwise, there is an issue with a video preview on the landing page:
<img width="1710" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/110097862/a1dca456-0187-4074-a6eb-dd768a7cadfd">

Reply:
Oh, I hadn't noticed.
I'll fix it.

Reply:
fixed

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/4d6bd538-1045-4ba7-94e4-c3e5a18db5b4)
right now it looks like this